pygeo/parameterization/DVGeo.py

```diff
@@ -1672,6 +1673,15 @@ def update(self, ptSetName, childDelta=True, config=None):
         # We've postponed things as long as we can...do the finalization.
         self._finalize()
 
+        for iChild in range(len(self.children)):
```

Comment:
Should we be adding this before we finalize the parent to ensure it has all of the required point-sets embedded?

Reply:
I don't think it matters since `_finalize` only computes the links for the FFD control points not the embed points.

Reply:
Got it. That makes sense! I misinterpreted what was happening there.